pkgs/applications/window-managers/sway/swayhide.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "swayhide";
+  version = "v0.2.1";
```

Comment:
This shouldn't have a v prefix. In fact I don't understand how your fetchFromGitHub works since it's also prepending a "v"...

Reply:
Reopening this comment thread: you did fix the double prepending, but we don't usually have a prefix for Nix package versions. Please move the "v" to the source fetching instead.